plugin/pkg/scheduler/generic_scheduler.go

```diff
@@ -166,20 +176,37 @@ func PrioritizeNodes(pod *api.Pod, podLister algorithm.PodLister, priorityConfig
 		priorityFunc := priorityConfig.Function
 		prioritizedList, err := priorityFunc(pod, podLister, minionLister)
 		if err != nil {
-			return algorithm.HostPriorityList{}, err
+			return schedulerapi.HostPriorityList{}, err
 		}
 		for _, hostEntry := range prioritizedList {
 			combinedScores[hostEntry.Host] += hostEntry.Score * weight
 		}
 	}
+	if len(extenders) != 0 && minionLister != nil {
+		nodes, err := minionLister.List()
+		if err != nil {
+			return schedulerapi.HostPriorityList{}, err
+		}
+
+		for _, extender := range extenders {
+			prioritizedList, err := extender.Prioritize(pod, &nodes)
+			if err != nil {
+				return schedulerapi.HostPriorityList{}, err
```

Comment:
I think in this case I think you should just "continue". Prioritization is by definition best-effort, so skipping priority extenders that generate an error seems fine. At least you will be able to use the built-in prioritization.